packages/bbui/src/Form/Core/CheckboxGroup.svelte

```diff
@@ -1,19 +1,24 @@
-<script>
+<script lang="ts" context="module">
+  type O = any
+  type V = any
+</script>
+
+<script lang="ts" generics="O, V">
```

Comment:
TIL about the `generics` attribute here.

Reply:
Yeah it's cool!

Slightly wonky because to make eslint happy about it you have to add that extra script block (https://stackoverflow.com/a/78981916), but I'm optimistic that's the sort of thing that will be improved on over time.